<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSetBase</class>
 <widget class="QDialog" name="DialogSetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>810</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Set Base</string>
  </property>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_get_base_image">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_set_base_image">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>550</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Set base image as camera input image.</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>600</y>
     <width>291</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Set Area</string>
   </property>
   <widget class="QPushButton" name="pushButton_set_detect_area">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Detect Area</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>80</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Mask Area</string>
    </property>
    <widget class="QPushButton" name="pushButton_mask_push">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Push</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_mask_pop">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Pop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_mask_clear">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_mask_enable_inside">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>231</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>enable inside</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_clear_detect_area">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Detect Area</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>291</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Zoom</string>
   </property>
   <widget class="QPushButton" name="pushButton_zoom_set">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_zoom_reset">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>30</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>291</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>NAME : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_name">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_name_change">
      <property name="text">
       <string>Change</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ID : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_id">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>40</y>
     <width>291</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Object</string>
   </property>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>271</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Feature</string>
    </property>
    <widget class="QSlider" name="horizontalSlider_feature_level">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>40</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>min</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>max</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Level :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_feature_level">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>221</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>-1:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_use_custom_feature">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Custom</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_select_object">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Object</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reset_object">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Object</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_select_erase">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Erase Feature</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reset_erase">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Erase</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>410</y>
     <width>291</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>Detect</string>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_detect_margin">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Margin :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Count :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_detect_threshold">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0.35</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_detect_count">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_detect_margin_set">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_detect_margin_get">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Get</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_detect_threshold_get">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Get</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_detect_threshold_set">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_detect_count_set">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_detect_count_get">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Get</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>271</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Constraint</string>
    </property>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>251</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Angle</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_constraint_angle_min">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>40</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>-180</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>30</y>
        <width>16</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>º</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>30</y>
        <width>16</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>º</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_constraint_angle_max">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>40</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>180</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_constraint_angle_set">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_constraint_angle_get">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Get</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>16</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>~</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>271</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Scale</string>
    </property>
    <widget class="QCheckBox" name="checkBox_scale_enable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_scale_min">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0.8</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_scale_max">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>70</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>1.2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>21</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>~</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_scale_precision">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0.01</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Precision :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>16</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>(</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>16</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_scale_set">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_scale_get">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Get</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_image_bg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>320</y>
     <width>291</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Set Reference Point</string>
   </property>
   <widget class="QPushButton" name="pushButton_ref_point_select">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Point</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ref_point_reset">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Point</string>
    </property>
   </widget>
  </widget>
  <zorder>label_image_bg</zorder>
  <zorder>label_image</zorder>
  <zorder>pushButton_get_base_image</zorder>
  <zorder>pushButton_set_base_image</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>groupBox_5</zorder>
  <zorder>groupBox_6</zorder>
  <zorder>groupBox_7</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
